shift/__manifest__.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2020 Coop IT Easy SCRL fs
+#   Elouan Le Bars <elouan@coopiteasy.be>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+
+{
+    "name": "Shift Management",
+    "summary": """Generate and manage shifts for cooperators.""",
+    "author": """
+        Thibault Francois,
+        Elouan Le Bars,
+        Coop IT Easy SC,
+        Odoo Community Association (OCA),
+        """,
```

Comment:
```suggestion
    "author": (
        "Thibault Francois, "
        "Elouan Le Bars, "
        "Coop IT Easy SC, "
        "Odoo Community Association (OCA)"
    ),
```

this should not be a multiline string, as it results in this:
```python
"\n        Thibault Francois,\n        Elouan Le Bars,\n        Coop IT Easy SC,\n        Odoo Community Association (OCA),\n        "
```

same comment for the other 2 modules.